EIPS/eip-3675.md

```diff
@@ -0,0 +1,247 @@
+
+### Total difficulty triggering the upgrade
+
+See [Security considerations](#terminal-total-difficulty-vs-block-number).
+
+### Halting the import of PoW blocks
+
+See [Security considerations](#halt-the-importing-of-pow-blocks).
+
+### Replacing block fields with constants
+
+Deprecated block fields are replaced with constant values to ensure the block format remains backwards compatible. Preserving the block format aids existing smart contracts and services in providing uninterrupted service during and after this transition.
+
+Particularly, this is important for those smart contracts that verify Merkle proofs of transaction/receipt inclusion and state by validating the hash of externally provided block header string against the corresponding value returned by the `BLOCKHASH` operation.
+
+This change introduces an additional validity rule that enforces the replacement of deprecated block fields.
+
+### Replacing `difficulty` with `0`
+
+After deprecating the proof-of-work the notion of difficulty no longer exists and replacing the block header **`difficulty`** field with `0` constant is semantically sound.
+
+### Deprecating `extraData`
+
+The **`extraData`** field is deprecated in favour of the corresponding **`graffiti`** field of the beacon block.
+
+### Changing block validity rules
+
+The rule set enforcing the PoW seal validity is replaced with the corresponding PoS rules along with the consensus upgrade as the rationale behind this change.
+
+An additional rule validating a set of deprecated block fields is required by the block format changes introduced by this specification.
+
+### Removing block rewards
+
+Existing rewards for producing and sealing blocks are deprecated along with the PoW mechanism. The new PoS consensus becomes both responsible for sealing blocks and for issuing block rewards once this specification enters into effect.
+
+### Supplanting fork choice rule
+
+The fork choice rule of the PoW mechanism becomes completely irrelevant after the upgrade and is replaced with the corresponding rule of the new PoS consensus mechanism.
+
+### Removing block gossip
+
+After the upgrade of the consensus mechanism only the beacon chain network will have enough information to validate a block. Thus, block gossip provided by the `eth` network protocol will become unsafe and is deprecated in favour of the block gossip existing in the beacon chain network. 
+
+It is recommended for the client software to not propagate descendants of any terminal PoW block to reduce the load on processing the P2P component and stop operating in the environment with unknown security properties.
+
+## Test Cases
+*TBD*
+
+## Reference Implementation
+*TBD*
+
```

Comment:
```suggestion
```
Consider removing these until you are ready to fill them in.  They are optional sections, and thus not required to be present to merge this as a draft and if they never get filled in that is technically acceptable.  Generally, we want to write EIPs in a way that is "ready to merge as final" without TODOs.